### PR TITLE
Feature/card breakdown chart

### PR DESCRIPTION
**Pull request by SirZach**
addresses #69

### CODE REVIEW

File: app/components/card-breakdown-chart.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import layout from '../templates/components/card-breakdown-chart';
+
+export default Ember.Component.extend({
+  layout: layout,
+
+  bars: function () {
+    var deck = this.get('deck');
+    var cardGroups = deck.get('cardGroups');
+    var mainCount = deck.get('mainCount');
+    var stats = {
```
Comment (ahaurw01) [inline app/components/card-breakdown-chart.js]: What do you think about generating these based on the `card.mainType`s that are present in the deck? One absence I can think of is Plainswalker. [Looks like](http://mtg.wikia.com/wiki/Card_Types) Tribal might be another type that could show up. Not really sure if that actually ever is used as a first class type.

File: app/components/card-breakdown-chart.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        width: 0
+      }
+    };
+
+    cardGroups
+      .filterBy('board', 'main')
+      .forEach((group) => {
+        var stat = stats[group.get('card.mainType')];
+        var count = group.get('count');
+        stat.count += count;
```
Comment (ahaurw01) [inline app/components/card-breakdown-chart.js]: I think this might bomb if the hard-coded types don't encompass what the deck contains.

Comment (SirZach) [inline app/components/card-breakdown-chart.js]: don't bring logic into this

File: app/styles/components/az-bars.scss [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  width: calc(100% - 96px);
+  div {
+    text-align: right;
+    padding: 5px;
+    height: 30px;
+    span {
+      width: 100%;
+      padding-right: 5px;
+      display: inline-block;
+      &.Creature {background-color: map-get($color-red, '300');}
```
Comment (ahaurw01) [inline app/styles/components/az-bars.scss]: The bar colors here could possibly be associated with card colors at first glance, but I do like the splash of color.

Comment (SirZach) [inline app/styles/components/az-bars.scss]: I was doing that for simple re-use of colors. I'm open to color suggestions. Also, what do you think of the az-bars tag?

File: app/components/card-breakdown-chart.js (hunk 2 above)
Comment (ahaurw01) [inline app/components/card-breakdown-chart.js]: lol

**Comment by ahaurw01** [comment]
I like it. At first I was thinking we should omit the zero value bars, but I now really like seeing that a deck decided to forego certain types. Aside from the thoughts above, :ship: it!

File: app/styles/components/az-bars.scss (hunk 3 above)
Comment (ahaurw01) [inline app/styles/components/az-bars.scss]: No brilliant thoughts here on color substitution. Making them all grey would probably be really bland. We can continue to think about it after merging.

What's the `az-` for?

Comment (SirZach) [inline app/styles/components/az-bars.scss]: Aaron and Zach!

Comment (ahaurw01) [inline app/styles/components/az-bars.scss]: Had an inkling :smile:

File: app/components/card-breakdown-chart.js (hunk 1 above)
Comment (SirZach) [inline app/components/card-breakdown-chart.js]: updated. what do you think of the approach now?

**Pull request merged**